source/NVDAObjects/JAB/__init__.py

```diff
@@ -86,18 +86,19 @@
 }
 
 JABStatesToNVDAStates={
-	"busy":controlTypes.State.BUSY,
-	"checked":controlTypes.State.CHECKED,
-	"focused":controlTypes.State.FOCUSED,
-	"selected":controlTypes.State.SELECTED,
-	"pressed":controlTypes.State.PRESSED,
-	"expanded":controlTypes.State.EXPANDED,
-	"collapsed":controlTypes.State.COLLAPSED,
-	"iconified":controlTypes.State.ICONIFIED,
-	"modal":controlTypes.State.MODAL,
-	"multi_line":controlTypes.State.MULTILINE,
-	"focusable":controlTypes.State.FOCUSABLE,
-	"editable":controlTypes.State.EDITABLE,
+	"busy": controlTypes.State.BUSY,
+	"checked": controlTypes.State.CHECKED,
+	"focused": controlTypes.State.FOCUSED,
+	"selected": controlTypes.State.SELECTED,
+	"pressed": controlTypes.State.PRESSED,
+	"expanded": controlTypes.State.EXPANDED,
+	"collapsed": controlTypes.State.COLLAPSED,
+	"iconified": controlTypes.State.ICONIFIED,
+	"modal": controlTypes.State.MODAL,
+	"multi_line": controlTypes.State.MULTILINE,
+	"focusable": controlTypes.State.FOCUSABLE,
+	"selectable": controlTypes.State.SELECTABLE,

```

Comment:
Can you minimize the diff so that only this line is added to the bottom of the set?
It makes reviewing this change simpler. We can lint the entire set in another PR.

Reply:
I have reverted my fix, applied just the fix in commit 38e09ca98672be19b91fb9513815cd45be1adb6f and then did linting of the dictionary in commit a04587b224e586f841aee3d356ea5814ac9b4805. Take what you will, I feel I have already spent long enough on such a small fix.

Reply:
The suggestion was to perform the linting in a separate PR (or not at all), and only perform the required change in this PR.